Added manual steps to user guide (#70)
exasol/transformers-extension: doc/user_guide/user_guide.md

```

#### Install Language Container
- To install the language container, it is necessary to load the container into the BucketFS 
and register it to the database. The following command provides this setup:

> ```buildoutcfg

Reviewer: "buildoutcfg" seems to be not in list of [markdown_supported_languages](https://github.com/jincheng9/markdown_supported_languages). Maybe you want to replace it by "shell"?

> You can install the language container in two different methods, described below:

Reviewer: ```suggestion
There are two ways to install the language container: (1) using a python script and (2) manual installation. See the next paragraphs for details.
```

     provides this setup using the python script provided with this library:

      ```buildoutcfg
      python -m exasol_transformers_extension.deploy language-container
          --dsn <DB_HOST:DB_PORT> \
          --db-user <DB_USER> \
          --db-pass <DB_PASSWORD> \
          --bucketfs-name <BUCKETFS_NAME> \
          --bucketfs-host <BUCKETFS_HOST> \
          --bucketfs-port <BUCKETFS_PORT> \
          --bucketfs-user <BUCKETFS_USER> \
          --bucketfs-password <BUCKETFS_PASSWORD> \
          --bucket <BUCKETFS_NAME> \
          --path-in-bucket <PATH_IN_BUCKET> \
          --language-alias <LANGUAGE_ALIAS> \ 
          --container-file <path/to/language_container.tar.gz>       
      ```

  2. *Manual Installation*

     In the manual installation, the pre-built container should be firstly 
     uploaded into BucketFS. In order to do that, you can use 
     either a [http(s) client](https://docs.exasol.com/database_concepts/bucketfs/file_access.htm) 
     or the [bucketfs-client](https://github.com/exasol/bucketfs-client). 
     The following command uploads a given container into BucketFS through curl 

>      command, a http(s) client: 

Reviewer: ```suggestion
     command, an http(s) client: 
```

      ```buildoutcfg
      curl -vX PUT -T \ 
          "<CONTAINER_FILE>" 

>           "http://w:<BUCKETFS_WRITE_PASS>@$bucketfs_host:<BUCKETFS_PASS>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"

Reviewer: ```suggestion
          "http://w:<BUCKETFS_WRITE_PASSWORD>@$bucketfs_host:$bucketfs_port/<BUCKETFS_NAME>/<PATH_IN_BUCKET>/<CONTAINER_FILE>"
```

      ```

> 

Reviewer: ````suggestion

Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `bucketfs_write_password`:
```shell 
read -sp "password: " bucketfs_write_password
```
````

      The uploaded container should be secondly activated through adjusting 

>       session parameter `SCRIPT_LANGUAGES`. The activating can be performed for 

Reviewer: ```suggestion
      the session parameter `SCRIPT_LANGUAGES`. The activation can be scoped
```

      either session-wide (`ALTER SESSION`) or system-wide (`ALTER SYSTEM`). 
      The following example query activates the container session-wide:

>       ```buildoutcfg

Reviewer: ```suggestion
      ```sql
```

      ALTER SESSION SET SCRIPT_LANGUAGES=\
      <ALIAS>=localzmq+protobuf:///<BUCKETFS_NAME>/<BUCKET_NAME>/<PATH_IN_BUCKET><CONTAINER_NAME>/?\
              lang=<LANGUAGE>#buckets/<BUCKETFS_NAME>/<BUCKET_NAME>/<PATH_IN_BUCKET><CONTAINER_NAME>/\
              exaudf/exaudfclient_py3
      ```
     

>       where `ALIAS` is _PYTHON3_TE_, `LANGUAGE` is _python_ in the 
>       transformers-extension project. For more details please check [Adding New Packages to Existing Script Languages](https://docs.exasol.com/database_concepts/udf_scripts/adding_new_packages_script_languages.htm).

Reviewer: ```suggestion
      In project transformer-extensions replace `<ALIAS>` by `_PYTHON3_TE_` and `<LANGUAGE>` by `_python_`.
      For more details please check [Adding New Packages to Existing Script Languages](https://docs.exasol.com/database_concepts/udf_scripts/adding_new_packages_script_languages.htm).
```

     or the [bucketfs-client](https://github.com/exasol/bucketfs-client). 
     The following command uploads a given container into BucketFS through curl 
     command, an http(s) client: 
      ```shell
      curl -vX PUT -T \ 
          "<CONTAINER_FILE>" 

>           "http://w:<BUCKETFS_WRITE_PASSWORD>@$bucketfs_host:<BUCKETFS_PORT>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"

Reviewer: ```suggestion
          "http://w:<BUCKETFS_WRITE_PASSWORD>@<BUCKETFS_HOST>:<BUCKETFS_PORT>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"
```

      ```

>       Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `bucketfs_write_password`:

Reviewer: ```suggestion
      Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `BUCKETFS_WRITE_PASSWORD`:
```
